### ISSUE TITLE
log of determinant for inverse pass

### ISSUE THREAD

**Issue by kaido975** [slimgroup/InvertibleNetworks.jl#70]
If I am not missing it, It would be great to have this feature.

**Comment by rafaelorozco** [comment]
Yes I agree, I have a branch with this implemented. I will open a PR later today. Will you mostly be using the GLOW network?

**Comment by rafaelorozco** [comment]
Ok, this branch https://github.com/slimgroup/InvertibleNetworks.jl/tree/glow-map implements the logdet on inverse and the backward_inv for glow network. The code base doesn't expect inverse to output logdet so some of the other network tests fail. But I made sure that the glow network and layers tests pass. I will fix the rest of the tests to merge to master in a bit but you are welcome to play around with that branch in the meanwhile.

**Comment by fabienbaron** [comment]
Looks like this has not been merged? Useful though.
